docs/get-started/using-with-mapbox-gl.md

```diff
@@ -3,7 +3,7 @@
 While a deck.gl application could choose to render on top of any JavaScript
 Map component (or even render without an underlying map),
 deck.gl has been developed in parallel with
-the [react-map-gl](https://uber.github.io/react-map-gl/#/) component,
+the [react-map-gl](https://github.com/uber/react-map-gl) component,
```

Comment:
Given the website is quite outdated, I preferred linking to the repo, where the readme docs are pretty complete